ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ptp.schema.yml

```diff
@@ -14,6 +14,8 @@ keys:
         valid_values:
         - boundary
         - transparent
+      boundary_one_step:
```

Comment:
Hmmm regarding the valid_values `transparent` for mode:

```
l3leaf1(config)#ptp mode ?
  boundary        Boundary Clock Mode
  disabled        Disable PTP and forward PTP messages normally
  e2etransparent  End-to-End Transparent Clock
  gptp            Generalized PTP Clock
  ordinarymaster  Ordinary Master Clock
  p2ptransparent  Peer-to-Peer Transparent Clock
```

@ClausHolbechArista maybe on another device (using CloudEOS here)

Reply:
Shovani did point this out to me, but that is unrelated to this pr.

Reply:
Unrelated though maybe let's open an issue to investigate further from this comment. Closing this.

Reply:
https://github.com/aristanetworks/avd/issues/3669